scalafix/input-0_14/src/main/scala/fix/v0_14_0/FixAvroCoder13.scala

```diff
@@ -0,0 +1,14 @@
+/*
+rule = FixAvroCoder
+ */
+package fix.v0_14_0
+
+import com.spotify.scio.ScioContext
+
+object FixAvroCoder13 {
+  val sc = ScioContext()
+
+  val a: A = ???
+
+  sc.parallelize(Seq(a))
```

Comment:
Note: I didn't want to add a separate file for each case, but i manually checked all permutations of `sc.parallelize(Seq(a))`, `sc.parallelize(Seq(new A))`, `val seqA = Seq(a); sc.parallelize(seqA)`, `val seqA = Seq(new A); sc.parallelize(seqA)`, etc...